### PR TITLE
Update setup.md

### PR DESCRIPTION
**Pull request by benjaminjosefsson**
Walk through to share whole screen.

### CODE REVIEW

**Review by Magnus93 — APPROVED** [review]

File: profile/setup.md [hunk 1]
```diff
@@ -109,3 +109,23 @@ npm install
...
 ```
+
+## Sharing Screen
+Sharing screen is disabled if your build uses wayland. To circumvent this do the following:
+Open terminal
+Check if system uses wayland:
+```
+echo $XDG_SESSION_TYPE
+```
+If the command outputs wayland continue with
```
Comment (Ztawh) [inline profile/setup.md:120]: If the command outputs "wayland" continue with

File: profile/setup.md [hunk 2]
```diff
@@ -109,3 +109,23 @@ npm install
...
+```
+sudo nano /etc/gdm3/custom.conf
+```
+and uncommit this line:
```
Comment (Ztawh) [inline profile/setup.md:124]: and uncomment this line:

File: profile/setup.md [hunk 3]
```diff
@@ -109,3 +109,23 @@ npm install
...
+```
+#WaylandEnable=false
+```
+Save and exit, reboot and checking with first command should display
```
Comment (Ztawh) [inline profile/setup.md:128]: Save, exit and then reboot the computer. If you check with first command again, it should display:

**Review by Ztawh — COMMENTED** [review]
Suggestions how to make it more clear.

**Review by simonmika — APPROVED** [review]

**Pull request merged**